forge-gui/res/cardsfolder/upcoming/harnessed_snubhorn.txt

```diff
@@ -0,0 +1,10 @@
+Name:Harnessed Snubhorn
+ManaCost:3 W
+Types:Creature Dinosaur
+PT:2/5
+K:Vigilance
+T:Mode$ DamageDone | ValidSource$ Card.Self | Optional$ True | ValidTarget$ Player | Execute$ TrigReturn | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, return target artifact or enchantment card from your graveyard to the battlefield.
```

Comment:
Optional here doesn't make sense but maybe you wanted to put it on the other card?

Reply:
Yeah